adapters/amqp-vertx/src/test/java/org/eclipse/hono/adapter/amqp/impl/VertxBasedAmqpProtocolAdapterTest.java

```diff
@@ -13,6 +13,7 @@
 package org.eclipse.hono.adapter.amqp.impl;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
can you please refactor the tests to use assertj instead?

Reply:
```
 assertThat(errorConditionCaptor.getValue().getCondition().toString())
                .isEqualTo(AmqpError.UNAUTHORIZED_ACCESS.toString());
```

```
 assertEquals(AmqpError.UNAUTHORIZED_ACCESS, 
                errorConditionCaptor.getValue().getCondition());
```
IMHO in the above context, `asserEquals(...)` suited better as there is no need to explicitly convert to string and then compare it as in the case of `asserThat(...)` as it doesn't allow `Symbol ` as an argument. Also we widely use `assertEquals(...)` in other tests too and hence I used it here too. I could also refactor it to use it to `assertThat(...)`. WDYT?

Reply:
I see. Let's keep it like this, then ...